app/javascript/plugins/fundraiser/FundraiserView.js

```diff
@@ -30,6 +53,23 @@ export class FundraiserView extends Component {
     }
   }
 
+  componentDidUpdate(prevProps) {
+    if (
+      prevProps.experiments.length == 0 &&
+      this.props.experiments.length > 0
+    ) {
+      const { variant } =
+        this.props.experiments.find(
+          e => (e.experimentId = unintendedDonationsExperiment.experimentId)
+        ) || {};
+      console.log(this.props);
```

Comment:
I added this for debugging/testing purposes, and I will remove it in a follow-up PR after testing it on staging.